CDP4Orm/AutoGenDao/ActualFiniteStateDao.cs

```diff
@@ -388,6 +483,31 @@ public override bool Delete(NpgsqlTransaction transaction, string partition, Gui
             return this.AfterDelete(beforeDelete, transaction, partition, iid);
         }
 
+        /// <summary>
+        /// Delete a database record from the supplied data transfer object.
+        /// A "Raw" Delete means that the delete is performed without calling BeforeDelete or AfterDelete.
+        /// This is typically used during the import of existing data to the Database.
+        /// </summary>
+        /// <param name="transaction">
+        /// The current <see cref="NpgsqlTransaction"/> to the database.
+        /// </param>
+        /// <param name="partition">
+        /// The database partition (schema) where the requested resource will be deleted.
+        /// </param>
+        /// <param name="iid">
+        /// The <see cref="CDP4Common.DTO.ActualFiniteState"/> id that is to be deleted.
+        /// </param>
+        /// <returns>
+        /// True if the concept was successfully deleted.
+        /// </returns>
+        public override bool RawDelete(NpgsqlTransaction transaction, string partition, Guid iid)
+        {
+            var result = false;
+
+            result = base.Delete(transaction, partition, iid);
```

Comment:
This could do directly:
```
return base.Delete(transaction, partition, iid);
```

Reply:
In this class, you are right. But there is a possibility that the RawDelete method contains a bit more code. Then the Result variable may indeed vary. From code generation point of view this is the best for now, otherwise the code generation code will be less readable.